ngsPETSc/utils/firedrake.py

```diff
@@ -174,6 +174,20 @@ def curveField(self, order, tol=1e-8):
                             newFunctionCoordinates.sub(2).dat.data[datIdx] = curvedPhysPts[i][j][2]
     return newFunctionCoordinates
 
+def splitToQuads(plex, dim, comm):
```

Comment:
For quad meshing: is there a way to get netgen to make a quad-native mesh? If not, is there a way to get netgen to make a quad-dominant mesh, and then only split the triangles? I worry this will make for poor-quality meshes.